crypto/init.c

```diff
@@ -658,3 +662,15 @@ int OPENSSL_atexit(void (*handler)(void))
 
     return 1;
 }
+
+void OPENSSL_fork_prepare(void)
+{
+}
+
+void OPENSSL_fork_parent(void)
+{
+}
+
+void OPENSSL_fork_child(void)
+{
```

Comment:
Can we get some BIG SCARY WARNINGs that only async-signal-safe functions are permitted in _parent and _child?